tests/fixtures/upgrade/resources.py

```
@@ -0,0 +1,193 @@
+        namespace=upgrade_namespace_scope_session,
+        client=admin_client,
+    ) as nad:
+        yield nad
+
+
+@pytest.fixture(scope="session")
+def running_vm_upgrade_a(
+    unprivileged_client,
+    upgrade_bridge_marker_nad,
+    kmp_enabled_namespace,
+    upgrade_br1test_nad,
```

Ruff (0.15.21)

[warning] Unused function argument: `upgrade_br1test_nad`

(ARG001)